osd.xml

```diff
@@ -245,16 +245,16 @@
             <font>small</font>
             <area>15,75,1534,45</area>
             <align>hcenter,bottom</align>
-            <template>%PLAYEDTIME% of %TOTALTIME% %(|REMAININGTIME|)%%(|BEHINDTIME|)%</template>
+            <template>%PLAYEDTIME% of %TOTALTIME% %(|REMAININGTIME| remaining)%%(|BEHINDTIME| behind)%</template>
```

Comment:
If this is not translated, then why do you want to add "remaining" rather than let the numbers stand alone?